### PR TITLE
Windows: Don't copy import .lib & .exp files for executables

### PR DESCRIPTION
**Pull request by kinke**
One aspect of issue #2377 - copy those for DLLs only.

### CODE REVIEW

**Comment by kinke** [comment]
Pinging @rikkimax.

**Comment by rikkimax** [comment]
Just received a not received email for GH (overloaded).  I said LGTM.

**Comment by kinke** [comment]
A test will be added in #2396.

File: source/dub/generators/build.d [hunk 1]
```diff
@@ -409,23 +409,23 @@ class BuildGenerator : ProjectGenerator {
...
+					filenames ~= candidate;
+			}
+
+			const tt = buildsettings.targetType;
+			if (tt == TargetType.executable || tt == TargetType.dynamicLibrary)
+				addIfFound(".pdb");
+
+			if (tt == TargetType.dynamicLibrary) {
+				addIfFound(".lib");
+				addIfFound(".exp");
```
Comment (Geod24) [inline source/dub/generators/build.d:427]: Could we use a `switch` here ?

**Review by Geod24 — APPROVED** [review]
LGTM, just one comment

File: source/dub/generators/build.d (hunk 1 above)
Comment (kinke) [reply source/dub/generators/build.d:427]: We definitely could, just not sure whether this is really better:
```D
switch (buildsettings.targetType) {
	case TargetType.dynamicLibrary:
		addIfFound(".lib");
		addIfFound(".exp");
		goto case;
	case TargetType.executable:
		addIfFound(".pdb");
		break;
	default: break;
}
```

**Pull request merged**